.Lib9c.Tools/SubCommand/Tx.cs

```diff
@@ -250,5 +251,16 @@ public void MigrationActivatedAccountsState()
             byte[] raw = _codec.Encode(encoded);
             Console.WriteLine(ByteUtil.Hex(raw));
         }
+
+        [Command(Description = "Create RenewAdminState action and dump it.")]
+        public void RenewAdminState(
+            [Argument("NEW-VALID-UNTIL")]
+            long newValidUntil
+        )
+        {
+            PolymorphicAction<ActionBase> action = new RenewAdminState(newValidUntil);
+            byte[] raw = _codec.Encode(action.PlainValue);
+            Console.WriteLine(ByteUtil.Hex(raw));
```

Comment:
We need to specify `nameof(RedeemAdminState)` as action type.

Reply:
I had misunderstood their logic 😞 I'll patch it now.

Reply:
I amended it in f3ecdde. Please review again! @longfin